cssfiles/cards.css

```diff
@@ -0,0 +1,66 @@
+.vertical-card{
+    background-color: var(--primary-color);
+    color: var(--primary-text-color);
+    box-shadow: 0 4px 8px 0 var(--primary-color);
```

Comment:
using variable for box shadow is recommended(so that you can use it elsewhere)

Reply:
ok will consider that